### PR TITLE
Added code for pact testing

### PR DESCRIPTION
**Pull request by SangeethaPonnusamy**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by yogendraU — APPROVED** [review]

**Review by hmcts-kodatala — APPROVED** [review]
OK

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -20,4 +21,13 @@
...
         <cve>CVE-2022-22971</cve>
     </suppress>
+  <suppress>
+    <notes>tomcat-embed-*</notes>
+    <cve>CVE-2022-34305</cve>
+  </suppress>
+  <suppress>
+  <notes>log4j-*</notes>
+  <cve>CVE-2022-33915</cve>
+  <cve>CVE-2022-33915</cve>
```
Comment (rahulsanjava19) [inline config/owasp/suppressions.xml:31]: please remove duplicate line

File: src/smokeTest/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          uri: '#{environment["TEST_URL"].replaceFirst("https://", "http://")}'
+
+#If you use a database then uncomment below lines and update db properties accordingly
+spring:
+  config:
+    import: "optional:configtree:/mnt/secrets/fis/"
+  application:
+    name: Fis Cos API
+  main:
+    allow-bean-definition-overriding: true
```
Comment (rahulsanjava19) [inline src/smokeTest/resources/application.yaml:36]: please remove commented lines

File: src/contractTest/java/uk/gov/hmcts/reform/cosapi/clients/ccd/CaseManagementConsumerTest.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+@PactTestFor(providerName = "ccd_submitForCitizen_api", port = "5001")
+@PactFolder("pacts")
+@SpringBootTest({  "ccd_submitForCitizen_api:http://localhost:5001"})
+public class CaseManagementConsumerTest {
+
+    private static final String SERVICE_AUTHORIZATION_HEADER = "ServiceAuthorization";
+    private static final String someServiceAuthToken = "someServiceAuthToken";
```
Comment (rahulsanjava19) [inline src/contractTest/java/uk/gov/hmcts/reform/cosapi/clients/ccd/CaseManagementConsumerTest.java:31]: these four variables along with consumer ="fis-cos-api" can be created as constants as these are used in most of the test classes

**Review by rahulsanjava19 — CHANGES_REQUESTED** [review]

**Review by rahulsanjava19 — APPROVED** [review]

**Pull request merged**